src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -310,6 +315,8 @@
                 "WHERE accountId = :accountId AND id = :id";
         String webhookQuery = "UPDATE WebhookProperties SET url = :url, method = :method, " +
                 "disableSslVerification = :disableSslVerification, secretToken = :secretToken WHERE endpoint.id = :endpointId";
+        String camelQuery = "UPDATE EndpointCamel SET url = :url, subType = :subType, " +
```

Comment:
Should this be `CamelProperties` instead of `EndpointCamel` ?

Reply:
Good catch and interesting case. I think it worked because of

`@Table(name = "endpoint_camel")` to with snake-case naming this could be seen as tablename = EndpointCamel which Hibernate perhaps just used as it did not find the right one. Pure speculation though